test/cli_network_test.go

```diff
@@ -285,6 +285,20 @@ func (suite *PouchNetworkSuite) TestNetworkCreateWithOption(c *check.C) {
 	defer command.PouchRun("network", "remove", funcname)
 }
 
+// TestNetworkCreateWithIPAMOption creates network with ipam options
+func (suite *PouchNetworkSuite) TestNetworkCreateWithIPAMOption(c *check.C) {
+	gateway := "192.168.100.1"
+	subnet := "192.168.100.0/24"
+	networkName := "TestNetworkCreateWithIPAMOption"
+	command.PouchRun("network", "create",
+		"--name", networkName,
+		"-d", "bridge",
+		"--gateway", gateway,
+		"--subnet", subnet,
+		"--ipam-opt", "test=foo").Assert(c, icmd.Success)
```

Comment:
how to check the ipam-opt is working？

Reply:
@rudyfly Done.

Reply:
The same question as rudyfly 👍

Reply:
@houstar I have updated the test codes for checking the return value of ipam-opt after network create.